### PR TITLE
reference fusion docs

### PR DESCRIPTION
**Pull request by timotheecour**
followup after https://github.com/nim-lang/fusion/pull/24 (EDIT)

pending https://github.com/nim-lang/fusion/pull/26 to make the link valid

fixes https://forum.nim-lang.org/t/6904

### CODE REVIEW

File: config/nimdoc.cfg [hunk 1]
```diff
@@ -132,7 +132,10 @@ doc.body_toc_group = """
         <a href="$theindexhref">Index</a>
       </li>
       <li>
-        <a href="compiler/$theindexhref">Compiler docs</a>
+        <a href="compiler/$theindexhref">compiler docs</a>
```
Comment (narimiran) [inline config/nimdoc.cfg:135]: Why the change of case here?

Comment (timotheecour) [reply config/nimdoc.cfg:135]: it's for consistency with `import compiler/foo` (instead of `import Compiler/foo`), ditto with `import fusion/bar` instead of `import Fusion/bar`

I don't care much either way here.

still though, I would've preferred to get  nim-lang/fusion#26  merged first as indicated in top post to make sure the link is valid

**Comment by alaviss** [comment]
Can we have this reverted until nim-lang/fusion#26 actually got merged? Also the link is very misleading. It would always refer to the latest devel docs, while the installed/bundled version might be different.

**Comment by alaviss** [comment]
Note that fusion docs are actually built and is in the stdlib `theindex.html`. The fix to get this out fast is to modify `kochdocs` to build `lib/fusion` as a separate project.

**Comment by timotheecour** [comment]
> Can we have this reverted until nim-lang/fusion#26 actually got merged?

it got merged finally

> the link is very misleading. It would always refer to the latest devel docs, while the installed/bundled version might be different.

we can change the link to fusion docs (master)


> Note that fusion docs are actually built and is in the stdlib theindex.html. The fix to get this out fast is to modify kochdocs to build lib/fusion as a separate project.

indeed, https://nim-lang.github.io/Nim/filepermissions.html is generated and should not be (at least not at that url).
That's a consequence of copying fusion into Nim/lib/fusion, which as I stated in https://github.com/nim-lang/fusion/issues/25 needs to be revisited or properly thought through, but we can improve things in the meantime.

see https://github.com/nim-lang/RFCs/issues/265 for a proposal on how to publish versioned docs for nim + fusion
